pages/irma-uitleg.md

```diff
@@ -305,6 +305,15 @@ update proces.
 
 ### <a name="architectuur"></a>5. Hoe verschilt IRMA van andere authenticatie systemen als iDIN en idensys?
 
+IRMA is geen authenticatie systeem, maar een identificatie systeem.
+Het is geen manier om te bewijzen wie je bent, maar om te laten zien
+wie je bent en dit is een wezenlijk ander proces. Wanneer je een
+gebruikersnaam en wachtwoord invoert dan: 1) claim je wie je bent ("username")
+en 2) bewijs je vervolgens dat jij dit bent ("wachtwoord"). Dit proces
+heet "authenticatie". Als je op een geloofwaardige manier laat zien
+wie je bent dan heet dit "identificatie". Dit is wat IRMA doet middels
+het onthullen van de attributen.
```

Comment:
Is de age check een vorm van een Zero-knownledge proof systeem? Zie: https://en.wikipedia.org/wiki/Zero-knowledge_proof

Reply:
Nee, ten tijde van uitgifte wordt aan de hand van de geboortedatum afgeleid of de ontvanger ouder dan 18 is, en dat wordt dan als normaal attribuut (met inhoud `Yes`/`Ja`) uitgegeven. Voor hoe IRMA zero-knowledge proofs gebruikt, zie [hier](https://irma.app/docs/overview/).